paimon-core/src/main/java/org/apache/paimon/utils/TagManager.java

```diff
@@ -69,6 +69,11 @@ public Path tagPath(String tagName) {
         return new Path(tablePath + "/tag/" + TAG_PREFIX + tagName);
     }
 
+    /** Return the path of tag directory in branch. */
+    public Path branchTagDirectory(String branchName) {
```

Comment:
Should this be used when counting tags or filter tags to distinguish the tags in default main branch and custom branches?

Reply:
Yes, assuming the current table directory is table, then the tags of default main branch are in table/tag, and the tags of custom branches are in table/custom_branch/tag. The branch to which the tag belongs depends on the location.